src/components/atoms/BooleanCell/BooleanCell.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import BooleanCell from './BooleanCell';
+
+
+test('renders off state if no booleanVal passed ', () => {
+    const tableRow = document.createElement('tr');
```

Comment:
I like the creation of the element the component is supposed to be wrapped in.

Reply:
Yeah, this was needed to supress the errors we were seeing in the watch task during the session